vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -65,11 +64,21 @@ class MSBuildTools {
 
     // Set platform toolset for VS 2019
     if (this.version === '16.0') {
+      const results =
+        child_process
+          .execSync(
+            '"%ProgramFiles(x86)%\\Microsoft Visual Studio\\Installer\\vswhere.exe" -requires Microsoft.VisualStudio.ComponentGroup.UWP.VC.v141 -property installationPath',
+          )
+          .toString()
+          .split(EOL)[0] + '\\MSBuild\\Microsoft\\VC\\v150\\';
```

Comment:
Where does `v150` come from?

Reply:
v150 seems to be code for "UWP". Confusing, I know, but this is just using what's already there.

Keith and I are looking at removing this but I'm keeping it for now for parity

---
In reply to: [335240297](https://github.com/microsoft/react-native-windows/pull/3434#discussion_r335240297) [](ancestors = 335240297)

Reply:
Oh, I see now. It actually refers to the VS version `15.0`, i.e. 2017.
For whatever reason, even if using VS 2019, the MSVC 14.1 (`v141`) targets get installed in a "Visual studio 15.0" folder.